sound/soc/sof/sof-priv.h

```diff
@@ -480,6 +490,18 @@ struct snd_sof_dev {
 	void *private;			/* core does not touch this */
 };
 
+/* SOF generic IPC data */
+struct snd_sof_ipc {
+	struct snd_sof_dev *sdev;
+
+	/* protects messages and the disable flag */
+	struct mutex tx_mutex;
+	/* disables further sending of ipc's */
+	bool disable_ipc_tx;
+
+	struct snd_sof_ipc_msg msg;
+};
+
```

Comment:
is this moved from somewhere else?
This patch has too many changes really.

Reply:
@plbossart yes, moved from ipc.c